test/suites/proxy.sh

```diff
@@ -0,0 +1,56 @@
+test_proxy_device() {
+  
+  MESSAGE="Proxy device test string"
+  LOCAL_TCP_PORT=1234
```

Comment:
Don't hardcode ports, this will fail on Jenkins.

You need to use the `local_tcp_port` shell function to get a new unused port for this.